### DISCUSSION TITLE
Support for Higher Baud Rates?

### DISCUSSION THREAD

**Discussion by digamesystems** [espressif/arduino-esp32#10103] [Q&A]
Hello!
This is probably a rookie question in this forum... :) 
Looking at Espressif's documentation, it appears that the ESP32 is capable of higher baud rates than those supported by the Arduino Serial class. (max=460800 baud) -- Apparently, in UART mode it's supposed to be capable of up to 5MBS. Has anyone explored this from the Arduino side? I realize over the USB interface this may be impractical but having the ESP talk to another device at high speed, do some processing and report back at a lower speed over UART would be great for a project I'm working on. 
Thanks for the great work! 
Regards, 
John

**Comment by SuGlider** [comment]
UART should work at high baud rate. I have tested it using Serial Monitor with 2Mbps.
